flask/app/participants.py

```diff
@@ -37,22 +37,33 @@
 def list_participants():
 
     # parsing query parameters
-    max_rows = request.args.get("max_rows", default=100)
-    starts_with = request.args.get("starts_with", default="", type=str)
-    starts_with = f"{starts_with}%"  # sql syntax
-    order_by_col = request.args.get("order", default="participant_id", type=str)
-    # need some default or we need an ifelse statement, one with order_by method and one without. AFAIK there is no 'default' parameter we can pass into order_by to get the default sql ordering scheme
+    limit = request.args.get("limit", default=10)
+    page = request.args.get("page", default=1)
+    order_by_col = request.args.get("order_by_by", default="participant_id", type=str)
+    filter_by_col = request.args.get("filter_by", default="participant_id", type=str)
+    filter_val = request.args.get("filter_val", default="", type=str)
+    filter_val = f"%{filter_val}%"
```

Comment:
This works but to support all the filters in the frontend tables, we need to support multiple filters. i.e. support any number of fields of the form `column_name=value` as long as it refers to a valid column name that can be read. We may need to change the form so non-strings work too and that might be a bit complicated to do generically vs just implementing specific cases for those columns here.